src/components/Patient/MedicationStatementList.tsx

```diff
@@ -14,60 +21,56 @@ import {
   TableHeader,
   TableRow,
 } from "@/components/ui/table";
-import {
-  Tooltip,
-  TooltipContent,
-  TooltipTrigger,
-} from "@/components/ui/tooltip";
+
+import { Avatar } from "@/components/Common/Avatar";
 
 import routes from "@/Utils/request/api";
 import query from "@/Utils/request/query";
 import { formatDateTime } from "@/Utils/utils";
+import {
+  MEDICATION_STATEMENT_STATUS_STYLES,
+  MedicationStatement,
+} from "@/types/emr/medicationStatement";
 
 interface MedicationStatementListProps {
   patientId: string;
+  className?: string;
+  isPrintPreview?: boolean;
 }
 
 interface MedicationRowProps {
-  statement: any;
+  statement: MedicationStatement;
```

Comment:
this should be the read spec right?

```suggestion
  statement: MedicationStatementRead;
```

Reply:
Done